### PR TITLE
SEP-6 Tests

### PR DESCRIPTION
**Pull request by JakeUrban**
So I got most of the tests from the SEP24 suite working for SEP6, but there are a few issues:

Requests to `/deposit` and `/witihdraw` may require custom fields per-anchor, which are described in the anchor's `/info` endpoint. Since we can't know what these values should be ahead of time, we're hardcoding the data to use on a per-anchor basis. Right now I only added the reference server's fields.

Additionally, SEP6 requests to `/deposit` and `/withdraw` don't guarantee the creation of a transaction, unlike SEP24. This means we can't deterministically create transactions that can then be tested. The result is that test coverage for SEP-6 is less than SEP-24, particularly for the `/transaction(s)` endpoints.

### CODE REVIEW

File: cases-SEP24/toml.test.js [hunk 1]
```diff
@@ -48,10 +48,6 @@ describe("TOML File", () => {
       }
     });
 
-    it("uses TRANSFER_SERVER_SEP0024", async () => {
```
Comment (msfeldstein) [inline cases-SEP24/toml.test.js:51]: why take these out?

File: server/jest-controller/list.js [hunk 2]
```diff
@@ -29,14 +29,31 @@ module.exports = async () => {
...
 
+      let testProject = process.env.PROJECT ? process.env.PROJECT : "SEP24";
+
       const unorderedTests = output
         .trim()
         .split("\n")
         .map((line) => {
-          const testName = path.basename(line).split(".test.js")[0];
-          if (orderedTests[testProject].includes(testName)) {
+          if (path.dirname(line) == `cases-${testProject}`) {
```
Comment (msfeldstein) [inline server/jest-controller/list.js]: Do we need this line? I was able to remove it previously by adding line 9 above does that not work for your use case?

**Review by msfeldstein — APPROVED** [review]

File: cases-SEP24/toml.test.js (hunk 1 above)
Comment (JakeUrban) [reply cases-SEP24/toml.test.js:51]: Because SEP-6 anchors won't have that TOML attribute

File: server/jest-controller/list.js (hunk 2 above)
Comment (JakeUrban) [reply server/jest-controller/list.js]: Oh I think that was a merging mistake, I'll remove it

File: cases-SEP24/toml.test.js (hunk 1 above)
Comment (msfeldstein) [reply cases-SEP24/toml.test.js:51]: but this is in cases-sep24

Comment (JakeUrban) [reply cases-SEP24/toml.test.js:51]: Ah crap

Comment (JakeUrban) [reply cases-SEP24/toml.test.js:51]: Ok, added it back, removed the same test in SEP6

**Pull request merged**